specification/ws.extensions/x-kaazing-idle-timeout/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-idle-timeout/no.data.sent.by.server.client.timeout/response.rpt

```diff
@@ -0,0 +1,33 @@
+accept tcp://127.0.0.1:8001
+accepted
+connected
+
+# Connection request
+read "GET /echo HTTP/1.1\r\n"
+read "Upgrade: websocket\r\n"
+read "Connection: Upgrade\r\n"
+read "Host: localhost:8001\r\n"
+read "Origin: http://localhost:8001\r\n"
+read "Sec-WebSocket-Key: ePYyZxvR5fDLh/oEugMjbA==\r\n"
+read "Sec-WebSocket-Version: 13\r\n"
+read "Sec-WebSocket-Extensions: x-kaazing-idle-timeout\r\n"
+read "\r\n"
+
+# 101 Response
+
+write "HTTP/1.1 101 Web Socket Protocol Handshake\r\n"
+write "Connection: Upgrade\r\n"
+write "Date: Fri, 31 Dec 1999 23:59:59 GMT\r\n"
+write "Sec-WebSocket-Accept" "TXCKJSpylVJ7v9ZGLGbt58vypzE=\r\n"
+write "Sec-WebSocket-Extensions: x-kaazing-idle-timeout; timeout=2000\r\n"
+write "Server: Kaazing Gateway\r\n"
+write "Upgrade: websocket\r\n"
+write "\r\n"
+
+read notify HANDSHAKE_COMPLETE
+
+# Client should timeout and close the connection - this doesn't happen
+
+read await TICK_TWO
```

Comment:
We should remove this line. This is because it makes this script dependent on the request script, and will make it hard or impossible to use just this (response) script in client tests. For the same reason we cannot put in "read await TICK_ONE". I made this change on my box and it has the added benefit of making the test pass!